### PR TITLE
Update QuestieQuestFixes.lua

### PR DESCRIPTION
**Pull request by Gogo1951**
Adding more items that start quests that drop off unique mobs or objects.

### CODE REVIEW

File: Database/Corrections/QuestieQuestFixes.lua [hunk 1]
```diff
@@ -947,5 +947,76 @@ function QuestieQuestFixes:Load()
         [9262] = { -- bad race data
             [QuestieDB.questKeys.requiredRaces] = 77,
         },
+        [6564] = {
+            [QuestieDB.questKeys.startedBy] = {{4802},ni,{16790}},
+        },
+
```
Comment (BreakBB) [inline Database/Corrections/QuestieQuestFixes.lua]:
```suggestion
```

File: Database/Corrections/QuestieQuestFixes.lua [hunk 2]
```diff
@@ -947,5 +947,76 @@ function QuestieQuestFixes:Load()
...
+        [7761] = {
+            [QuestieDB.questKeys.startedBy] = {{9046},ni,{18987}},
+        },
+        [3181] = {
+            [QuestieDB.questKeys.startedBy] = {{5833},ni,{10000}},
+        },
+        [2] = {
+            [QuestieDB.questKeys.startedBy] = {{12676},ni,{16305}},
+        },
+
```
Comment (BreakBB) [inline Database/Corrections/QuestieQuestFixes.lua]:
```suggestion
```

**Comment by BreakBB** [comment]
Hey @Gogo1951 thanks for taking the time to submit this PR!

It would be great if you:

- Sort the corrections by ID, as the others are as well
- Share some screenshots for the quests you added (to see if they're not too much)

**Comment by BreakBB** [comment]
Oh and also it is `nil` and not `ni`.

**Comment by Gogo1951** [comment]
> Oh and also it is `nil` and not `ni`.

Yup. https://github.com/AeroScripts/QuestieDev/issues/1573#issuecomment-563202985 (=

**Comment by BreakBB** [comment]
Argh, my bad then ^^

**Comment by BreakBB** [comment]
I think this PR is obsolete because of #1616 ?

**Comment by Gogo1951** [comment]
Yes, obsolete. (=

**Pull request closed without merging**